src/app/apps/rup/mapa-camas/sidebar/cama-detalle/internacion-detalle/internacion-detalle.component.ts

```diff
@@ -108,17 +111,19 @@ export class InternacionDetalleComponent implements OnInit, AfterViewChecked {
         // Configura los tabs a mostrar según capa y vista
         this.mapaCamasService.resumenInternacion$.pipe(
             map(resumen => {
-                if (!!this.editar && this.editar !== resumen.paciente?.id) {
+                if (!!this.editarIngreso && this.editarIngreso !== resumen.paciente?.id) {
```

Comment:
Aca se podria evaluar la condición !!this.editarIngreso por this.editarIngreso. Además si el tipo de editarIngreso es boolena, condice compararlo con un tipo de dato distinto como lo es un id?

Reply:
totalmente, corregido!